### DISCUSSION TITLE
`corfu--auto-complete-deferred` ... ``

### DISCUSSION THREAD

**Discussion by kaibagley** [minad/corfu#387] [Q&A] [answered]
OS: Windows 10
Emacs version: 29.1 build 2
Corfu version: 0.38
ESS version: 18.10.3snapshot

Hey,

Please let me know if I should post this on the ESS github instead, but when using corfu + ESS, I regularly get stuttering while typing, and if I stop typing at the right time I can see in the minibuffer an error message:

> `Error running timer 'corfu--auto-complete-deferred': (user-error "ESS process not ready. Finish your command before trying again")`

It is quite hard to to work with as typing a letter during a stutter will cause it to not appear in the buffer. Let me know if you'd like me to report on anything else.

**Comment by minad** [comment, marked as answer]
Yes, please report this on the ESS tracker. I assume that the problem is that the ESS Capf is not interruptible.

**Reply by kaibagley** [reply]
Sure, thanks for the reply!
